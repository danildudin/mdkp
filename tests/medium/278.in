30 8
568 491 912 743 611 483 634 349 803 628 411 606 238 34 109 368 83 685 883 440 824 768 819 792 859 706 399 901 2 216 
859 246 362 345 524 175 796 662 274 211 63 229 158 621 903 878 706 582 261 596 553 548 262 110 639 204 544 614 46 672 7926
705 697 229 114 101 128 831 875 33 676 986 358 577 55 138 214 706 80 746 296 937 371 816 246 736 684 66 84 921 650 5775
248 447 632 558 719 638 958 464 579 565 121 921 764 527 112 982 731 73 455 318 582 755 807 200 421 781 131 190 137 488 7662
472 477 467 198 870 778 733 487 611 149 442 602 629 9 578 24 639 685 646 173 410 235 227 979 666 793 700 747 851 897 5668
981 90 158 341 913 16 444 918 198 378 195 723 628 394 37 773 193 776 773 733 788 806 183 402 226 387 721 493 928 363 7173
465 653 857 142 518 125 497 108 638 152 810 878 508 744 717 118 809 620 22 243 743 756 336 136 750 395 867 466 823 892 5266
166 883 534 246 603 557 386 167 873 369 597 370 563 333 693 665 721 21 379 996 9 210 588 561 58 993 859 33 797 4 7924
43 756 358 941 367 814 939 949 250 626 852 846 769 706 169 730 247 996 931 771 71 213 614 75 129 59 474 872 65 440 5560
