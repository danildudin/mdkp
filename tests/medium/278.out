9049
12
0 2 3 8 9 15 18 20 21 23 24 27 
